astropy/table/pprint.py

```diff
@@ -384,8 +383,11 @@ def _pformat_col_iter(self, col, max_lines, show_name, show_unit, outs,
         n_print2 = max_lines // 2
         n_rows = len(col)
 
-        col_format = col_getattr(col, 'format')
-        format_func = _format_funcs.get(col_format, _auto_format_func)
+        col_format = col.info.format
+        if col_format is None and hasattr(col.info, 'default_format_func'):
```

Comment:
With my comments on changing to `default_format`, this would become
```
col_format = col.info.format or getattr(col.info, 'default_format', None)
format_func = _format_funcs.get(col_format, _auto_format_func)
```